templates/style.css

```diff
@@ -0,0 +1,514 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
Is this file redundant with `static/lib/style.css`? Please consider removing one

Reply:
One of the CSS file is for testing (the one in templates) and the other CSS file (the one in static) is for when running flask. For some reason for me to open and test in default browser, I have to have the CSS within the local folder. And for flask it only registers the file in the static folder.